src/metadata_utils.jl

```diff
@@ -65,11 +65,16 @@ function _extend!(program::Expr, trait::Symbol, value, T)
     end
 end
 
-const DEPWARN_DOCSTRING =
-    "`metadata_model` should not be called with the keyword argument "*
-    "`descr` or `docstring`. Implementers of the MLJ model interface "*
-    "should instead create an MLJ-compliant docstring in the usual way. "*
-    "See https://alan-turing-institute.github.io/MLJ.jl/dev/adding_models_for_general_use/#Document-strings for details. "
+const depwarn_docstring(T) =
```

Comment:
is there any need for the `const` keyword. Function defined in global space of a module is already defined as a constant.

Reply:
Nope. Removed.